starwars/src/components/ListComponent.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Character from './CharacterComponent';
+
+
+const CharacterList = (info) => {
+    return ( <
+            div >
+            <
+            h1 > Characters < /h1> {
+            info.character.map(character => {
+                return <
+                    Character i = { character.created }
```

Comment:
What you're doing here is sending the Character component a prop defined as i = {character.created}. This works, but it will only send the time and date that the character was created to the Character component. We don't only want to send that single key value to the Character component - we want to send to the Character component a prop that contains all the key values associated with the character object. Like so: 

```Character i = { character}```